apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -260,7 +265,15 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
     /**
      * Load application template data.
      */
-    const loadTemplateDetails = (id: string): void => {
+    const loadTemplateDetails = (id: string, template: ApplicationTemplateInterface): void => {
+
+        // If the loading strategy is `LOCAL`, stop making the GET request to fetch template details.
+        if (templateLoadingStrategy === ApplicationTemplateLoadingStrategies.LOCAL) {
+            setTemplateSettings(template);
+
+            return;
+        }
```

Comment:
```suggestion
    }
```

Reply:
This curly brace terminates the if block.